alf/algorithms/td_loss.py

```diff
@@ -198,3 +214,121 @@ def _summarize(v, r, td, suffix):
         # [T, B], so we need to augment it with additional zeros.
         loss = tensor_utils.tensor_extend_zero(loss)
         return LossInfo(loss=loss, extra=loss)
+
+
+@alf.configurable
+class TDQRLoss(TDLoss):
+    """Temporal difference quantile regression loss. 
+    Compared to TDLoss, GAE support has not been implemented. """
+
+    def __init__(self,
+                 num_quantiles=50,
+                 gamma=0.99,
+                 td_error_loss_fn=losses.huber_function,
+                 td_lambda=1.0,
+                 sum_over_quantiles=False,
```

Comment:
I think it is better to set the default value of `sum_over_quantiles ` to be `True` as per the QR-DQN paper (the last line of Algorithm 1, https://arxiv.org/pdf/1710.10044.pdf)

Reply:
Updated.